easybuild/easyconfigs/m/Mesa/Mesa-25.1.3-GCCcore-14.2.0.eb

```
@@ -0,0 +1,65 @@
+# This is a Mesa using software rendering via Gallium-DRI and libglvnd
+# - libglvnd can dynamically choose between system-installed NVidia
+# libGLX/libEGL or the software renderers provided by this Mesa
+# - EGL is available
+#
+# Software renderers enabled (swr deprecated as of v22):
+# - llvmpipe: uses LLVM for JIT code generation (multi-threaded)
+# - softpipe: a reference Gallium driver
+# Default renderer is llvmpipe. To use softpipe, set the environment
+# variable GALLIUM_DRIVER=softpipe
+
+name = 'Mesa'
+version = '25.1.3'
+
+homepage = 'https://www.mesa3d.org/'
+description = """Mesa is an open-source implementation of the OpenGL specification -
+ a system for rendering interactive 3D graphics."""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+source_urls = [
+    'https://archive.mesa3d.org/',
+    'https://archive.mesa3d.org/older-versions/%(version_major)s.x/',
+]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['ffcb6cadb5fd356d56008e6308641dfe4b2929f30139f6585436ca6e3cddba7f']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Meson', '1.6.1'),
+    ('Ninja', '1.12.1'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.8.2'),
+    ('pkgconf', '2.3.0'),
+    ('Mako', '1.3.10'),
+    ('libxml2', '2.13.4'),
+    ('expat', '2.6.4'),
+    ('gettext', '0.24'),
```

```suggestion
    ('gettext', '0.24'),
    ('glslang', '15.3.0'),
    ('PyYAML', '6.0.2'),
```

Atleast for me both packages were needed to compile

Will need gsllang EC from
- https://github.com/easybuilders/easybuild-easyconfigs/pull/23171